requirements.txt

```diff
@@ -1,17 +1,21 @@
 aniso8601==4.1.0
 asn1crypto==0.24.0
-atomicwrites==1.2.1
+atomicwrites==1.3.0
 attrs==18.2.0
 aws-xray-sdk==0.95
+Babel==2.6.0
 backcall==0.1.0
+blinker==1.4
 boto==2.49.0
-boto3==1.9.86
-botocore==1.12.86
+boto3==1.7.84
+botocore==1.10.84
```

Comment:
remove the boto libraries, It's already included in rudra and you are not using it anywhere.